src/vm/stats.rs

```diff
@@ -66,5 +82,35 @@ pub mod stats_view_model {
                 soulsmemory
             }
         }
+
+        pub fn from_nya_model(nyam: &nya::Model) -> Self {
+            let arche_type = ArcheType::try_from(nyam.character_class.clone()).expect("");
+            let vigor = nyam.stats.vigor;
+            let mind = nyam.stats.mind;
+            let endurance = nyam.stats.endurance;
+            let strength = nyam.stats.strength;
+            let dexterity = nyam.stats.dexterity;
+            let intelligence = nyam.stats.intelligence;
+            let faith = nyam.stats.faith;
+            let arcane = nyam.stats.arcane;
+            let level = nyam.stats.level;
+            let souls = Default::default();
```

Comment:
These are set to default until Emilia can make an update to include the calculated values.